crates/sc-subspace-block-relay/src/protocol/compact_block.rs

```diff
@@ -146,11 +146,13 @@ where
     }
 
     /// Fetches the missing entries from the server
-    async fn resolve_misses(
+    async fn resolve_misses<
+        RequestMsg: From<CompactBlockRequest<DownloadUnitId, ProtocolUnitId>> + Encode + Send + Sync,
```

Comment:
Please write explicit `where` to avoid such long and hard to read signatures in function definition.

Reply:
Done